esphome/packages/sensors/rgb_led.yaml

```diff
@@ -1,4 +1,8 @@
 # https://esphome.io/components/light/neopixelbus.html
+substitutions:
+  strobe_on: 50ms
```

Comment:
Please prefix your substitutions with `rgb_led_effects_` and add them also as a comment to the `esphome.yaml`.